audit.json

```diff
@@ -227,6 +228,9 @@
   "10054_Cookie Without SameSite Attribute_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/create-order/about-to-start_POST": "ignore",
   "10054_Cookie Without SameSite Attribute_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/return-application/about-to-start_POST": "ignore",
   "10054_Cookie Without SameSite Attribute_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/review-cmo/about-to-submit_POST": "ignore",
+  "10054_Cookie Without SameSite Attribute_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/upload-c2/validate/mid-event_POST": "ignore",
+  "10054_Cookie Without SameSite Attribute_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/draft-standard-directions/populate-date-of-issue/mid-event_POST": "ignore",
+  "10054_Cookie Without SameSite Attribute_http://fpl-case-service-aat.service.core-compute-aat.internal/callback/draft-standard-directions/upload-route/mid-event_POST": "ignore";
```

Comment:
Invalid character I think at the end of line

Reply:
fixed